### PR TITLE
models - data class

### PR DESCRIPTION
**Pull request by chezi-nikop**

### CODE REVIEW

File: src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.hibob.academy.employeeFeedback.dao
+
+import java.time.LocalDate
+
+data class FeedbackInsert(
+    val employeeId: Long?,
+    val isAnonymous: Boolean,
```
Comment (elijulian) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt:7]: maybe we don't need this? like if the employee id isn't present then we know that it's anonymous, no?

File: src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package com.hibob.academy.employeeFeedback.dao
+
+import java.time.LocalDate
+
+data class FeedbackInsert(
```
Comment (elijulian) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt:5]: if you are already dividing it into two classes, let's be consistent with the naming convention - FeedbackIn, FeedbackOut

**Review by elijulian — APPROVED** [review]

File: src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    MANAGER,
+    EMPLOYEE;
+
+    companion object {
+        fun enumToString(role: RoleType): String =
+            role.toString()
+
+        fun stringToEnum(string: String): RoleType =
+            valueOf(string.uppercase())
+    }
```
Comment (elijulian) [inline src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt:47]: don't think you really need this, but if you like, sure 😄

**Review by elijulian — APPROVED** [review]

File: src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt (hunk 3 above)
Comment (chezi-nikop) [reply src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt:48]: Im using it in the mapper

**Pull request merged**